arbitrum-docs/run-arbitrum-node/nitro/03-nitro-database-snapshots.md

```diff
@@ -0,0 +1,148 @@
+INFO [08-22|18:10:55.297] Writing cached state to disk             block=39987 hash=ddcd60..fe0fc3 root=d6973e..7b9265
+INFO [08-22|18:10:55.298] Persisted trie from memory database      nodes=34  size=11.19KiB  time="283.875µs" gcnodes=0   gcsize=0.00B     gctime=0s          livenodes=2414 livesize=794.81KiB
+INFO [08-22|18:10:55.298] Writing cached state to disk             block=39861 hash=2a9dd3..f00ff0 root=139d5a..d6bf21
+INFO [08-22|18:10:55.298] Persisted trie from memory database      nodes=73  size=24.88KiB  time="502.916µs" gcnodes=0   gcsize=0.00B     gctime=0s          livenodes=2341 livesize=769.93KiB
+INFO [08-22|18:10:55.299] Writing cached state to disk             block=39861 hash=2a9dd3..f00ff0 root=139d5a..d6bf21
+INFO [08-22|18:10:55.299] Persisted trie from memory database      nodes=0   size=0.00B     time="1.417µs"   gcnodes=0   gcsize=0.00B     gctime=0s          livenodes=2341 livesize=769.93KiB
+INFO [08-22|18:10:55.299] Writing snapshot state to disk           root=bd18ce..3b0763
+INFO [08-22|18:10:55.299] Persisted trie from memory database      nodes=0   size=0.00B     time="1.125µs"   gcnodes=0   gcsize=0.00B     gctime=0s          livenodes=2341 livesize=769.93KiB
+INFO [08-22|18:10:55.304] Blockchain stopped
+```
+
+After nitro stopped, go to the database directory and generate an archive file the directories `arbitrumdata`, `l2chaindata`, and `nodes`. By default, the database directory for nitro is `$HOME/.arbitrum/$CHAIN/nitro`. The commands below exemplify how to generate the snapshot for Nitro.
+
+```shell
+export CHAIN=sepolia-rollup
+export ARCHIVE_PATH=/tmp/archive.tar.gz
+cd $HOME/.arbitrum/$CHAIN/nitro
+tar zcfv $ARCHIVE_PATH arbitrumdata l2chaindata nodes
+```
+
+This command purposely omits the `wasm` directory from the snapshot archive. The `wasm` contains native-code executable, so it might be a security concern for users downloading the snapshot. If the user downloading the snapshot trusts you, or if you are storing the snapshot for your own use, you may include the `wasm` directory in it.
+
+### Optional: divide it into parts
+
+It is possible to divide the snapshot into smaller parts to facilitate its download. This is particularly useful for archive snapshots of heavily used chains, such as arb1. These kind of snapshots can reach terabytes, so is useful to divide them into smaller parts. The snippet below illustrates how to divide the snapshot into parts using the split command. The `-b` argument tells split to divide the snapshot into 100 GB parts. The `-d` argument tells split to enumerate the parts using a numeric suffix instead of a alphabetic one.
+
+```shell
+split -b 100g -d archive.tar.gz archive.tar.gz.part
+```
+
+After dividing it into parts, you should generate the manifest file that contains the names and checksums of the parts. This files will be used by Nitro to know how many parts there are and to validate their checksum. The command below exemplifies how to do that.
```

Comment:
```suggestion
This command purposely omits the `wasm` directory from the snapshot archive. The `wasm` contains native-code executables, so it might be a security concern for users downloading the snapshot. If the user downloading the snapshot trusts you, or if you are storing it for your own use, you may include the `wasm` directory in it.

### Optional: divide it into parts

It is possible to divide the snapshot into smaller parts to facilitate its download. This is particularly useful for archive snapshots of heavily used chains, such as arb1. These kinds of snapshots can reach terabytes, so dividing them into smaller parts is helpful. The snippet below illustrates how to divide the snapshot into parts using the split command. The `-b` argument tells the split to divide the snapshot into 100 GB parts. The `-d` argument tells split to enumerate the parts using a numeric suffix instead of an alphabetic one.

```shell
split -b 100g -d archive.tar.gz archive.tar.gz.part
```

After dividing it into parts, you should generate the manifest file containing the parts' names and checksums. Nitro will use these files to know how many parts there are and to validate their checksum. The command below exemplifies how to do that.
```